final_prep/annotations.rb

```diff
@@ -19,25 +22,27 @@ def build_a_bear(name, age, fur, clothes, special_power)
   return built_bear
 end
 
-build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
+# Create an instance of the function with the arguments in parentheses.
+p build_a_bear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares')
```

Comment:
Here, we're not creating an instance of the function - we're **invoking** or _calling_ (running) the function.